.eslintrc.js

```diff
@@ -0,0 +1,57 @@
+module.exports = {
+	env: {
+		browser: true,
+		es2021: true,
+	},
+	parserOptions: {
+		ecmaVersion: '2021',
+	},
+	globals: {
+		App: 'readonly',
+		jQuery: 'readonly',
+		_: 'readonly',
+		$: 'readonly',
+	},
+	overrides: [
+		{
+			files: [
+				'web/assets/js/fx/*.js',
```

Comment:
use `web/assets/js/**/*.js` or add `web/assets/js/pages/*.js`

Reply:
Oh I forgot pages. I tried the whole js folder but it tries to lint the minified bundle, the framework and all the libs, that's why I added individual folders.